src/protocollogin.cpp

```diff
@@ -53,7 +53,7 @@ void ProtocolLogin::getCharacterList(const std::string& accountName, const std::
 	}
 
 	// Generate and add session key
-	static std::independent_bits_engine<std::default_random_engine, CHAR_BIT, unsigned char> rbe;
+	static std::independent_bits_engine<std::default_random_engine, CHAR_BIT, unsigned short> rbe;
```

Comment:
Note: the standard says "the effect is undefined unless the parameter is cv-unqualified and is one of `unsigned short`, `unsigned int`, `unsigned long`, or `unsigned long long`."

GCC and Clang accept `unsigned char`, but MSVC refuses to compile with an error message.